<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SmartDataDialog</class>
 <widget class="QWidget" name="SmartDataDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>702</width>
    <height>499</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SMART Data - ?</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLineEdit" name="searchLineEdit">
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Search key ...</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTreeWidget" name="treeWidget">
     <property name="contextMenuPolicy">
      <enum>Qt::ActionsContextMenu</enum>
     </property>
     <property name="iconSize">
      <size>
       <width>5</width>
       <height>5</height>
      </size>
     </property>
     <column>
      <property name="text">
       <string notr="true">Key</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Value</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Drive status</string>
       </property>
       <property name="margin">
        <number>5</number>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="statusLineEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="exportButton">
       <property name="text">
        <string>Export Data</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionCollapse_All">
   <property name="text">
    <string>Collapse All</string>
   </property>
  </action>
  <action name="actionExpand_All">
   <property name="text">
    <string>Expand All</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
